### PR TITLE
fix/JSON Serialisation when message decryption failed

### PR DESCRIPTION
**Pull request by mohitpubnub**
fix: json serialisation failed for custom objects when crypto module is configured.

Fixes issue of getting exception for custom objects in subscription and history when crypto module is configured.

### CODE REVIEW

File: src/Api/PubnubApi/Security/SecureMessage.cs [hunk 1]
```diff
@@ -68,12 +68,10 @@ public List<object> HistoryDecodeDecryptLoop<T>(PNOperationType type, List<objec
                             {
                                 status.AffectedChannelGroups.AddRange(channelGroups);
                             }
-
-                            errorCallback.OnResponse(default(T), status);
```
Comment (Xavrax) [inline src/Api/PubnubApi/Security/SecureMessage.cs:72]: Ohhh, I missed this during the previous change ;o?

**Review by Xavrax — APPROVED** [review]
LGTM

**Comment by mohitpubnub** [comment]
@pubnub-release-bot release as v6.19.5.0

File: src/Api/PubnubApi/Security/SecureMessage.cs (hunk 1 above)
Comment (mohitpubnub) [reply src/Api/PubnubApi/Security/SecureMessage.cs:72]: this is fine!

**Pull request merged**